src/main/java/no/sikt/nva/validators/ExcelRowValidator.java

```diff
@@ -0,0 +1,112 @@
+package no.sikt.nva.validators;
+
+import static no.sikt.nva.scrapers.ExcelScraper.CRISTIN_ID_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.EMBARGO_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.FILENAME_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.FIRST_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.LICENCE_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.TITLE_COLUMN;
+import static no.sikt.nva.scrapers.ExcelScraper.VERSION_COLUMN;
+import static no.sikt.nva.validators.ExcelHeaderValidator.VALID_HEADERS;
+import no.sikt.nva.brage.migration.common.model.record.BrageVersion;
+import no.sikt.nva.brage.migration.common.model.record.license.BrageLicense;
+import no.sikt.nva.exceptions.ExcelException;
+import nva.commons.core.StringUtils;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.DateUtil;
+import org.apache.poi.ss.usermodel.Row;
+
+public class ExcelRowValidator {
+    public static final String ERROR_MESSAGE_INVALID_NUMBER_OF_COLUMNS = "Invalid number of columns";
+    public static final String ERROR_MESSAGE_MISSING_FIRST_COLUMN_VALUE = "Missing first column value";
+    public static final String ERROR_MESSAGE_MISSING_CRISTIN_ID = "Missing Cristin Id";
+    public static final String ERROR_MESSAGE_INVALID_CRISTIN_ID = "Invalid Cristin Id";
+    public static final String ERROR_MESSAGE_MISSING_TITLE = "Missing Title";
+    public static final String ERROR_MESSAGE_MISSING_VERSION = "Missing Version";
+    public static final String ERROR_MESSAGE_INVALID_VERSION = "Invalid Version value";
+    public static final String ERROR_MESSAGE_INVALID_EMBARGO_FORMAT = "Invalid Embargo format";
+    public static final String ERROR_MESSAGE_MISSING_LICENCE = "Missing Licence";
+    public static final String ERROR_MESSAGE_INVALID_LICENCE = "Invalid Licence value";
+    public static final String ERROR_MESSAGE_MISSING_FILENAME = "Missing Filename";
+
+    public static void validateRow(Row row) throws ExcelException {
```

Comment:
Instead of making this void, make it return some object ex. "UnisContentFile" or a better name if you can come up with it. And let that object decide if the parameters to it are good enough.